### DISCUSSION TITLE
How do I make a link to an about blank?

### DISCUSSION THREAD

**Discussion by Jay12312323123123** [UseInterstellar/Interstellar#818] [Q&A] [answered]
I've tried to make a GitHub but once I make the codespace and type npm and stuff when I join the link it gets blocked by my school. How do I make it an about blank so they don't block it?

**Comment by 0mhx** [comment]
what's your blocker and how does the block screen look like?

**Reply by Jay12312323123123** [reply]
> what's your blocker and how does the block screen look like?

securely screen

**Reply by 0mhx** [reply, marked as answer]
about:blank will still be blocked for you then, unfortunately.
